### ISSUE TITLE
Extend for custom linters?

### ISSUE THREAD

**Issue by chevinbrown** [sds/slim-lint#112]
Is slim-lint extendable like rubocop or haml-lint?

I'd like to write a custom cop for internal use and the only way I've figured out how to include the correct linter in the registry is to put `slim-lint` in our binstubs and require the new linter explicitly there...

**Comment by sds** [comment]
You can in theory `require` a file before running Slim-Lint, but this isn't yet supported via a command line flag.

You could do something like:

```ruby
#!/usr/bin/env ruby
require 'my/slim/linter'
require 'slim_lint/cli'
logger = SlimLint::Logger.new(STDOUT)
exit SlimLint::CLI.new(logger).run(ARGV)
```

...and save this as an executable file. Note: I haven't tried this.

Would also welcome a pull request adding support for custom linters. Thanks!

**Comment by chevinbrown** [comment]
@sds that's exactly what we've done. We're doing a little work for custom-cops, but they're stretch-goals right now (not prioritized). 
This would be a pretty cool addition, maybe we can find time to do a PR after we've fixed up internal cops like #89.
